web_src/less/themes/theme-arc-green.less

```diff
@@ -68,6 +68,14 @@
   --color-black: #1e222e;
   --color-gold: #a1882b;
   --color-white: #ffffff;
+  --color-red-sha-bg: rgba(219, 40, 40, .1);
+  --color-red-sha-hover: rgba(219, 40, 40, .3);
+  --color-orange-sha-bg: rgba(242, 113, 28, .1);
+  --color-orange-sha-hover: rgba(242, 113, 28, .3);
+  --color-yellow-sha-bg: rgba(251, 189, 8, .1);
+  --color-yellow-sha-hover: rgba(251, 189, 8, .3);
+  --color-green-sha-bg: rgba(33, 186, 69, .1);
+  --color-green-sha-hover: rgba(33, 186, 69, .3);
```

Comment:
Please copy the remaining vars in here as well, I'd prefer this list to stay in sync, even if colors are the same.